frontend/app/routes/_gcweb-app.tsx

```diff
@@ -190,28 +159,11 @@ function PageHeader() {
                 </button>
                 {showDropdown && (
                   <div id="dropdownNavbar" className="text-deep-blue-dark z-10 rounded-b-[5px] bg-white drop-shadow-[0_4px_4px_rgba(0,0,0,0.25)] sm:absolute sm:w-[260px]" aria-labelledby="dropdownLargeButton">
-                    {menuOptions.map((element, index) => {
-                      return (
-                        <div key={element.id}>
-                          <Link
-                            className={`${index === 0 ? 'border-none' : 'border-t-2'} font-body hover:text-blue-hover ring-blue-hover flex h-[55px] items-center rounded-sm px-4 ring-offset-2 focus:border-none focus:outline-none focus:ring-2`}
-                            onClick={() => setShowDropdown((currentState) => !currentState)}
-                            to={element.path}
-                            aria-label={element.value}
-                          >
-                            {element.showIcon && (
-                              <svg width="18" height="15" viewBox="0 0 18 15" fill="none" xmlns="http://www.w3.org/2000/svg" className="mr-3">
-                                <path
-                                  d="M13.1665 3.33333L11.9915 4.50833L14.1415 6.66667H5.6665V8.33333H14.1415L11.9915 10.4833L13.1665 11.6667L17.3332 7.5L13.1665 3.33333ZM2.33317 1.66667H8.99984V0H2.33317C1.4165 0 0.666504 0.75 0.666504 1.66667V13.3333C0.666504 14.25 1.4165 15 2.33317 15H8.99984V13.3333H2.33317V1.66667Z"
-                                  fill="#284162"
-                                />
-                              </svg>
-                            )}
-                            {element.value}
-                          </Link>
-                        </div>
-                      );
-                    })}
+                    <MenuOption onClick={() => setShowDropdown((currentState) => !currentState)} href={t('gcweb:header.menu-dashboard.href', { baseUri: SCCH_BASE_URI })} i18nKey={t('gcweb:header.menu-dashboard.text')} />
+                    <MenuOption onClick={() => setShowDropdown((currentState) => !currentState)} href={t('gcweb:header.menu-profile.href', { baseUri: SCCH_BASE_URI })} i18nKey={t('gcweb:header.menu-profile.text')} />
+                    <MenuOption onClick={() => setShowDropdown((currentState) => !currentState)} href={t('gcweb:header.menu-security-settings.href', { baseUri: SCCH_BASE_URI })} i18nKey={t('gcweb:header.menu-security-settings.text')} />
+                    <MenuOption onClick={() => setShowDropdown((currentState) => !currentState)} href={t('gcweb:header.menu-contact-us.href', { baseUri: SCCH_BASE_URI })} i18nKey={t('gcweb:header.menu-contact-us.text')} />
+                    <MenuOption onClick={() => setShowDropdown((currentState) => !currentState)} icon={signOutIcon()} href={t('gcweb:header.menu-dashboard.href', { baseUri: SCCH_BASE_URI })} i18nKey={t('gcweb:header.menu-sign-out.text')} />
```

Comment:
A small suggestion.. You can pull out the click handler to avoid repetition:

``` typescript
const onClickHandler = () => setShowDropdown((currentState) => !currentState);
//
<MenuOption onClick={onClickHandler} />
```